### ISSUE TITLE
Depth to Disparity Transformation and Training Process

### ISSUE THREAD

**Issue by SoniaShona** [LiheYoung/Depth-Anything#175]
Hello,
Thank you for your valuable contribution. 
I am currently reproducing the training code and performing fine-tuning on pseudo-labeled data as described in the article. I have two questions:

1. In section 3.1, you mention that "the depth value is first transformed into the disparity space by d = 1/t." Is this formula applied to all public datasets you use, or does it depend on the dataset (for example: for the TartanAir dataset, the disparity formula is disp = 80/depth)?
2. In section 4.1, you mention: "In the second stage of joint training, we train a student model to sweep across all unlabeled images for one time." Does this mean that we pass through all the pseudo-labeled data only once per epoch, as in standard training, or that the student model is trained for only one epoch in total?

Thank you in advance for your response.

**Comment by LiheYoung** [comment]
Hi Reffad,
1. By default, we transform depth to disparity by ``d = 1/t``. Although TartanAir provides the formula ``d = 80 / t``, it indeed does not matter in our framework, because we will further normalize the disparity map by min-max value to 0-1, which will diminish the effect of the scalar ``80``.
2. The latter case. The student model is only trained on all the 62M unlabeled images for one epoch.
